src/main/resources/lang.yml

```diff
@@ -109,6 +110,7 @@ list-reward: "Reward(s):"
 list-reward-money: "receive AMOUNT"
 list-reward-item: "receive AMOUNT ITEM"
 list-reward-command: "other"
+list-reward-command-custom-message: "MESSAGE"
```

Comment:
Should we actually be giving users this new option? I think that having a custom `Display` option already gives them all the freedom they need, how would this additional configuration option prove useful? Having more options tends to be confusing and makes things more complex, I tend to only add them when they're really needed.

Reply:
Yeah. There isn't a a lot of usage cases. Sounds fine.